DOCKER.md

```diff
@@ -0,0 +1,63 @@
+# Cross compiling with `Docker`
+The manual process kind of pollutes your workstation by setting lots of environment variables and writing to various config files. Thus it can be difficult to remove or upgrade these files and settings or even repeat that process for different versions of rust on the same machine.
```

Comment:
This is a bit exaggerated, to be honest. The "unclean" part would the `gcc-sysroot` script + the config files. The `cross` scripts are meant to take care of and contain the environment variables, to leave the system untouched. So, the parts about various config files and difficulty to remove, upgrade and change version are at least correct.

What I'm trying to say is that there is no real reason to paint the manual method as worse than it is.